### DISCUSSION TITLE
SP633

### DISCUSSION THREAD

**Discussion by homeassistantjunkie** [monty68/uniled#76] [Q&A]
I am new to HA and I am experiencing few issues that I can't resolve.. It either HA is very amateurly made (Which I doubt) or I am too dumb to understand it..

I already made a few mistakes that I can't undo and it is really sad I can't figure out/bypass them..

I want to start with HA BT integration..
I got myself DIGITUS DN-30210-1 and as soon as I pugged it in my SFF detected it and I was able to use it without any issues. (First I bought a cheap 10$ BT adapter that HA was not able to recognize, when I figured there is no way of manually integrating it, I bought Digitus BT adapter which costed me around 30$)

I integrated HA UniLED v2.2.5 - The Universal Light Controller as I have 3-4 cheap LED strip controllers, models are SP633E and application that drives them is BanlanX.

Per UniLed description, this is a supported adapter but for some reason, I can only toggle it on/off and change from static to dynamic. I really want to be able to control light colors. Have in mind that via BanlanX, I am able to to do so much fun stuff, but fort HA I am only seeking option to change color of the light..

If any SS is needed, let me know, I am not that tech savy..

Please help!

**Comment by monty68** [comment]
Resolved in v2.2.6
